template.rb

```diff
@@ -84,6 +96,45 @@
 end
 CODE
 
+if yes?("Are you hosting Shipit on Heroku? (y/n)")
+  inject_into_file "Procfile", " -p $PORT", after: "web: bundle exec rails s"
+  inject_into_file "Gemfile", "ruby '2.2.3'", after: "source 'https://rubygems.org'\n"
+
+  gsub_file 'Gemfile', "# Use sqlite3 as the database for Active Record", ''
+  gsub_file 'Gemfile', "gem 'sqlite3'", ''
+  gem_group :production do
+    gem 'pg'
+    gem 'rails_12factor'
+  end
+  gem_group :development, :test do
+    gem 'sqlite3'
+  end
+
+  remove_file 'config/database.yml'
+  create_file 'config/database.yml', <<-EOF
```

Comment:
The only difference here is that the default database.yml file will have the entry:
```
production:
  <<: *default
  database: db/production.sqlite3
```

I was not sure if people typically use sqlite for the production db. Ill make that change if your ok with it changing away from having the database file selected.